processing/src/test/java/io/druid/segment/filter/ExtractionDimFilterTest.java

```diff
@@ -160,7 +160,7 @@ public ExtractionType getExtractionType()
   @Test
   public void testEmpty()
   {
-    ExtractionFilter extractionFilter = new ExtractionFilter(
+    SelectorFilter extractionFilter = new SelectorFilter(
```

Comment:
This should still be an ExtractionFilter because this test is meant to be testing ExtractionFilters. It should keep doing that until they are actually removed.